src/modules/data/blocks/ticket/sagas.js

```diff
@@ -223,11 +265,15 @@ export function* fetchTicket( action ) {
 			] );
 		}
 	} catch ( e ) {
+		console.error( e) ;
 		/**
 		 * @todo handle error scenario
 		 */
 	} finally {
-		yield put( actions.setTicketIsLoading( blockId, false ) );
+		const allIds = yield select( selectors.getAllTicketIds );
+		if ( allIds.indexOf( blockId ) !== -1 ) {
+			yield put( actions.setTicketIsLoading( blockId, false ) );
```

Comment:
Consider removing this outside the finally and removing the blockId check. This way, this can run without having to check if the id exists and will not run if return is called.

Reply:
Still missing this here @mitogh